runtime/onert/core/include/ir/Shape.h

```diff
@@ -25,6 +25,12 @@
 #include <vector>
 #include <algorithm>
 
+#ifdef UNSPECIFIED_DIM
+#error UNSPECIFIED_DIM is already defined
+#endif
+
+#define UNSPECIFIED_DIM -1
+
```

Comment:
@hyunsik-yoon What about defining in `struct Shape`?

```c++
struct Shape
{
public:
  static const int32_t UNSPECIFIED_DIM = -1;
```

Reply:
That would be better. Thanks. :-)